libraries/AP_L1_Control/AP_L1_Control.cpp

```diff
@@ -515,3 +519,34 @@ void AP_L1_Control::update_level_flight(void)
 
     _data_is_stale = false; // status are correctly updated with current waypoint data
 }
+
+// update L1 control for path following
+void AP_L1_Control::update_path(const class Location &position_on_path, Vector2f unit_path_tangent, float path_curvature, int8_t direction) {
```

Comment:
add comment on units, check sign of curvature
explain what zero means
explain what direction means